libs/npm/core/src/hooks/api/constants/ListOption.ts

```diff
@@ -0,0 +1,4 @@
+export enum ListOption {
+  Source = 0,
```

Comment:
Enum values are serialized as string values for the most part.  You'll need to create an additional enum `ListOptionName`